examples/README.md

```diff
@@ -0,0 +1,8 @@
+### TensorFlow Decision Forests (TF-DF)
+
+
+| Tutorial                                    | Description                                                                                                 |
+|---------------------------------------------|-------------------------------------------------------------------------------------------------------------|
+| [minimal](./minimal.py)                     | display and evaluate a Random Forest model on the adult dataset                                             |
+| [beginner_diabetes](./beginner_diabetes.py) | Beginner-friendly  usage example of TensorFlow Decision Forests (TF-DF) using pima India's Diabetes dataset |
```

Comment:
Could you please shorten the name (e.g. display and evaluate a Random Forest model on the pima India's diabetes dataset).

On that note, it would of course be great if you could rewrite the example using a different model (i.e. gradient boosted trees)

Reply:
Right now I'm shortening the description but will add another notebook using the same dataset for gradient boosted trees